core/bessctl.cc

```diff
@@ -1241,10 +1327,10 @@ class BESSControlImpl final : public BESSControl::Service {
 
     if (is_any_worker_running()) {
       propagate_active_worker();
-      if (m1->num_active_workers() || m2->num_active_workers() ) {
-	WorkerPauser wp; // Only pause when absolutely required
-	ret = m1->ConnectModules(ogate, m2, igate);
-	goto done;
+      if (m1->num_active_workers() || m2->num_active_workers()) {
+        WorkerPauser wp;  // Only pause when absolutely required
+        ret = m1->ConnectModules(ogate, m2, igate);
+        goto done;
       }
```

Comment:
(general comment: I like having the whitespace fixes as a separate commit - maybe we should just get that one in and done with first)

Reply:
Ah, whichever commit touched these lines last must not have been clang-formated correctly. We have a pre-commit hook for this, but it only really works if everyone installs it :)